core/src/main/scala/sttp/client3/sse/ServerSentEvent.scala

```diff
@@ -0,0 +1,31 @@
+package sttp.client3.sse
+
+import scala.util.Try
+
+case class ServerSentEvent(
+    data: Option[String] = None,
+    eventType: Option[String] = None,
+    id: Option[String] = None,
+    retry: Option[Int] = None
+)
+
+object ServerSentEvent {
+  def parseEvent(event: List[String]): ServerSentEvent = {
```

Comment:
maybe add some tests?
it's not possible for parsing to fail - there are no invalid inputs?